cmake/DLAF_AddTest.cmake

```diff
@@ -194,10 +147,25 @@ function(DLAF_addTest test_target_name)
     list(APPEND _TEST_ARGUMENTS ${_HPX_EXTRA_ARGS_LIST})
   endif()
 
+  ### Test executable target
+  add_executable(${test_target_name} ${DLAF_AT_SOURCES})
+  target_link_libraries(${test_target_name}
+    PRIVATE
+      ${_gtest_tgt}
+      DLAF_test
```

Comment:
Shouldn't you add something like this?

`$<$<BOOL:${IS_AN_MPI_TEST}>:MPI::MPI_CXX>`

Reply:
MPI is always linked because of `dlaf.prop`, we shouldn't need this.